students/kevin_t/lesson7/html_render.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+
+"""
+A class-based system for rendering html.
+"""
+
+
+# This is the framework for the base class
+class Element(object):
+    tag = "html"
+    indent = "    "
+    def __init__(self, content=None, **kwargs):
+        self.kw_dict = kwargs
+
+        if content is None:
+            self.contents = []
+        elif isinstance(content, list) == False:
```

Comment:
use `is False` or simply `elif not isinstance`

`==` is used with primitive data types and `is` is used with objects